tests/integration/targets/appliance/tasks/main.yml

```diff
@@ -2,7 +2,6 @@
 - import_tasks: appliance_access_dcui.yml
 - import_tasks: appliance_access_shell.yml
 - import_tasks: appliance_access_ssh.yml
-- import_tasks: appliance_health.yml
```

Comment:
maybe for now don't delete this test from the appliance, we will deal with zuul later

Reply:
I mean the existing tests aren’t running on Zuul

Reply:
yes, however, if for some reason we would like to enable them, we need them to exists still IMO
when we will finish all the test cases modification, we will think on deletion